modular/blocksyncer/blocksyncer_options.go

```diff
@@ -256,13 +263,15 @@ func (b *BlockSyncerModular) quickFetchBlockData(ctx context.Context, startHeigh
 			latestBlockHeightAny := Cast(b.parserCtx.Indexer).GetLatestBlockHeight().Load()
 			latestBlockHeight := latestBlockHeightAny.(int64)
 			if latestBlockHeight == int64(endBlock) {
-				continue
+				RealTimeStart.Store(true)
+				CatchEndBlock.Store(int64(endBlock))
+				return
 			}
 			if latestBlockHeight > int64(count*(cycle+1)+startHeight-1) {
 				startBlock = count*cycle + startHeight
 				endBlock = count*(cycle+1) + startHeight - 1
-				processedHeight := Cast(b.parserCtx.Indexer).ProcessedHeight
 				flag = 1
+				processedHeight := Cast(b.parserCtx.Indexer).ProcessedHeight
```

Comment:
Is there any special meaning to move processedHeight down one line here?

Reply:
Sorry, no special significance, ignore this line change